doc/manual.html

```diff
@@ -3156,6 +3162,122 @@ <h2 id="addons">Addons</h2>
     looked up through <a href="#getMode"><code>getMode</code></a>.</p>
 </section>
 
+<section id="vimapi">
+    <h2>VIM Mode API</h2>
+
+    <p>CodeMirror has a robust VIM mode that attempts to faithfully 
+    emulate VIM's most useful features. It can be enabled by 
+    including <a href="../keymap/vim.js"><code>keymap/vim.js</code>
+    </a> and setting the <code>keymap</code> option to 
+    <code>vim</code>.</p>
+
+    <h3 id="vimapi_configuration">Configuration</h3>
+
+    <p>VIM mode accepts configuration options for customizing
+    behavior at run time. These methods can be called at any time
+    and will affect all existing CodeMirror instances unless
+    specified otherwise. The methods are exposed on the
+    <code><strong>CodeMirror.Vim</strong></code> object.</p>
+
+    <dl>
+      <dt id="vimapi_setOption"><code><strong>setOption(name: string, value: any, ?cm: CodeMirror, ?cfg: object)</strong></code></dt>
+      <dd>Sets the value of a VIM option. <code>name</code> should
+      be the name of an option. If <code>cfg.scope</code> is not set
```

Comment:
Added a comment to `defineOption` to clarify that it's currently not possible to define a buffer-local option that does not have a global value. I mostly didn't add it to the current API because I'm planning to change the signature a bit in the next major release.